### PR TITLE
create search bar

### PR DESCRIPTION
**Pull request by LBondarchuk**
- [DEMO LINK](https://LBondarchuk.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://LBondarchuk.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,16 +1,50 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```
Comment (polosanya) [inline src/index.html:4]:
```suggestion
<html lang="en">
  <head>
```
Don't add newlines between parent and child elements

File: src/index.html [hunk 2]
```diff
@@ -1,16 +1,50 @@
...
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
     <h1>Search bar airbnb</h1>
```
Comment (polosanya) [inline src/index.html:14]:
```suggestion
```
It's absent on the design. You can remove it

File: src/index.html [hunk 3]
```diff
@@ -1,16 +1,50 @@
...
+    <form 
+      class="search" 
+      data-qa="big" 
+      action="http://localhost:8080" 
+      method="post"
+    >
+      <label>
+        <img alt="icon" class="icon" src="images/Search.svg">
```
Comment (polosanya) [inline src/index.html:22]:
```suggestion
        <img 
          alt="icon" 
          class="icon" 
          src="images/Search.svg"
        >
```
Every attribute should be placed on a new line if their amount >= 3

File: src/index.html [hunk 4]
```diff
@@ -1,16 +1,50 @@
...
+        <input 
+          id="id"
+          name="input"
+          type="text" 
+          class="input" 
+          placeholder=" Try &quot; Los Angeles&quot" data-qa="keypress"
+        >
```
Comment (polosanya) [inline src/index.html:29]:
```suggestion
          placeholder=" Try &quot; Los Angeles&quot" 
          data-qa="keypress"
        >
```

File: src/index.html [hunk 5]
```diff
@@ -1,16 +1,50 @@
...
+      </label>
+    </form>
+    <form 
+      class="search" 
+      data-qa="small" 
+      action="http://localhost:8080" 
+      method="post"
+    >
+      <label>
+        <img alt="icon--small" class="icon icon--small" src="images/Search.svg">
```
Comment (polosanya) [inline src/index.html:39]: Same problem

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+  position: absolute;
+  left: 26px;
+  top: 26px;
+}
+
+.icon--small {
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
```
Comment (polosanya) [inline src/style.css:69]: Add newline at the end
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

**Review by polosanya — CHANGES_REQUESTED** [review]
Tests link doesn't work. Please fix it.

File: readme.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://LBondarchuk.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://LBondarchuk.github.io/layout_search-bar-airbnb/report/html_report/)
```
Comment (Anastasiia-Svintsova) [inline readme.md:4]: Your test report link doesn't work. Did you run tests? 
image

File: src/index.html (hunk 2 above)
Comment (Anastasiia-Svintsova) [reply src/index.html:11]: Not fixed 
image

File: src/index.html [hunk 8]
```diff
@@ -2,15 +2,56 @@
...
+      action="http://localhost:8080" 
+      method="post"
+    >
+      <label>
+        <img 
+          alt="icon" 
+          class="icon" 
+          src="images/Search.svg"
+        >
+        <input
```
Comment (Anastasiia-Svintsova) [inline src/index.html:24]: Don't forget to add an empty line between multilane children
```suggestion
        >
        
        <input 
```

File: src/index.html [hunk 9]
```diff
@@ -2,15 +2,56 @@
...
+          id="id"
+          name="input"
+          type="text" 
+          class="input" 
+          placeholder=" Try &quot; Los Angeles&quot" 
+          data-qa="keypress"
+        >
+      </label>
+    </form>
+    <form
```
Comment (Anastasiia-Svintsova) [inline src/index.html:34]:
```suggestion
    </form>
    
    <form 
```

File: src/index.html [hunk 10]
```diff
@@ -2,15 +2,56 @@
...
+      method="post"
+    >
+      <label>
+        <img 
+          alt="icon--small" 
+          class="icon 
+          icon--small" 
+          src="images/Search.svg"
+        >
+        <input
```
Comment (Anastasiia-Svintsova) [inline src/index.html:47]:
```suggestion
        >
        
        <input 
```

File: src/index.html [hunk 11]
```diff
@@ -2,15 +2,56 @@
...
+      class="search" 
+      data-qa="small" 
+      action="http://localhost:8080" 
+      method="post"
+    >
+      <label>
+        <img 
+          alt="icon--small" 
+          class="icon 
+          icon--small"
```
Comment (Anastasiia-Svintsova) [inline src/index.html:44]:
```suggestion
          class="icon icon--small" 
```

File: src/index.html [hunk 12]
```diff
@@ -2,15 +2,56 @@
...
+    <form 
+      class="search" 
+      data-qa="big" 
+      action="http://localhost:8080" 
+      method="post"
+    >
+      <label>
+        <img 
+          alt="icon" 
+          class="icon"
```
Comment (Anastasiia-Svintsova) [inline src/index.html:21]: Don't you need to add modifiers for the big input as well? 
```suggestion
          class="icon icon--big" 
```

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+  box-shadow: 0px 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 3px;
+}
+        
+.icon {
+  width: 19px;
+  z-index: 1;
+  position: absolute;
+  left: 26px;
+  top: 26px;
```
Comment (Anastasiia-Svintsova) [inline src/style.css:62]: Better to move styles for the big input to another selector. .icon--big for example

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]

**Pull request closed without merging**